internal/repos/importer.go

```diff
@@ -84,13 +84,15 @@ var (
 )
 
 type githubOrgRepoImportSettings struct {
-	Org                string `json:"org"`
-	RemoveDeletedRepos bool   `json:"removeDeletedRepos"`
+	Org                string   `json:"org"`
+	RemoveDeletedRepos bool     `json:"removeDeletedRepos"`
+	DefaultSyncs       []string `json:"defaultSyncs"`
```

Comment:
@ramiroJCB could you please share an example config JSON for this. Are you going to be using repo_sync_types here? If so, I think we should call it types here as well.

Reply:
Yeah , my suggestion was that we show the user in a way our available syncs types to them and the pick.
So yeah  you like **DefaultSyncTypes**  or just **SyncTypes**

Reply:
DefaultSyncTypes makes sense to me.